### ISSUE TITLE
Event script for mouse and keyboard mainly for phones with broken screens.

### ISSUE THREAD

**Issue by DienoX** [Genymobile/scrcpy#3558]
- [ ] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Is your feature request related to a problem? Please describe.**
Unable to control the phone with broken screen and adb turned off.

**Describe the solution you'd like**
Add the ability to create input event macros that you save to a file. This will allow you to create a macro on your phone with adb enabled. Then you can run this macro on the phone without adb enabled to run this option. 

**Describe alternatives you've considered**
Ready preset with the ability to play scripts.

**Additional context**

**Comment by navid-zamani** [comment]
This is already possible with the operating system’s facilities.  
You can simulate arbitrary input on Linux and for X. And record arbitrary input.  
`scrcpy --otg` would simply map your system’s input devices to simulated USB OTG ones for the phone. Nothing else.

In its most fundamental and elegant form, for Linux and other Unix-likes, remember that everything is a file. So you just `cat /dev/input/event$deviceNo > ~/myInput`, and then later, while the program is active, `cat ~/myInput > /dev/input/event$deviceNo`, where `$deviceNo` is the number, gathered from looking at `ls -la /dev/input/by-*`.

Window systems like X, which scrcpy of course uses, make this more complicated. Because they already read all events. So the above won’t work.  
They have their own tools for this (like `xev` and `xdotool`) of course, but they are more cumbersome and don’t offer the elegance of just reading from and writing to a file. There probably are tools for that too, but they are of course outside of the scope of `scrcpy` and should not be duplicated functionality inside of `scrcpy` either. :D

TL;DR: On Linux/Unix, `scrcpy --otg` already offers all you need. Everything else is just standard OS input recording/simulation.
